src/NuGet.Services.AzureSearch/SearchDocumentBuilder.cs

```diff
@@ -361,18 +363,86 @@ private static void PopulateKey(KeyedDocument document, string packageId, Search
             document.IsExcludedByDefault = isExcludedByDefault;
         }
 
-        private static void PopulateDeprecation(
+        private static void PopulateDeprecationFromDb(
             SearchDocument.Full document,
             Package package)
+        {
+            if ((int)package.Deprecations?.FirstOrDefault()?.Status > (int)PackageDeprecationStatus.NotDeprecated)
```

Comment:
How about change to 
```suggestion
            if (package.Deprecations?.FirstOrDefault()?.Status != PackageDeprecationStatus.NotDeprecated)
```
It more readable and easier to understand if condition for package deprecated.